tests/test_builders/test_build_epub.py

```diff
@@ -169,19 +182,21 @@ def test_build_epub(app):
 
     # nav.xhtml / nav
     navlist = nav.find('./xhtml:body/xhtml:nav')
-    toc = navlist.findall('./xhtml:ol/xhtml:li')
```

Comment:
This and other tests had the same issue - `toc` was defined as an  `EPUBElementTree` and here is being redefined as a `list[EPUBElementTree]`. I changed the variable names to avoid redefining with a different type.